server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -132,12 +132,6 @@ public class ShuffleServerConf extends RssBaseConf {
           .withDescription(
               "Expired time (ms) for application which has no heartbeat with coordinator");
 
-  public static final ConfigOption<Integer> SERVER_MEMORY_REQUEST_RETRY_MAX =
```

Comment:
Why do we remove this config option? Do we need to modify the document at the same time?

Reply:
We have already switched from _server retry_ to _client retry_. We don't need this server configuration anymore.

Reply:
OK. Do we need to modify the description of client retry config option?

Reply:
> OK. Do we need to modify the description of client retry config option?

Currently, we don't have descriptions for client retry configs. I think it's more like internal configs. 

And also, our configurations are quite irregular, without a unified format. Some are defined in File A, some in File B. Some types are ConfigEntry, some are ConfigOption. Some have descriptions in the README, some only in the Java files, and some don't even have descriptions in the Java files, like the current client retry configuration. I feel that we should standardize the configuration part in the future, fill in the missing information, improve the README, and so on. I think these series of operations should be completed in a separate PR. The current PR is only focusing on issue https://github.com/apache/incubator-uniffle/issues/1606.

Reply:
OK,
Servers use different config options from clients. Different frameworks will have different config options.